src/guide/essentials/template-syntax.md

```diff
@@ -225,7 +225,7 @@ Similarly, you can use dynamic arguments to bind a handler to a dynamic event na
 <a v-on:[eventName]="doSomething"> ... </a>
 
 <!-- shorthand -->
-<a @[eventName]="doSomething">
+<a @[eventName]="doSomething"> ...</a>
```

Comment:
Nice catch.

Could you add in the extra space, to keep it consistent with the other examples?
```suggestion
<a @[eventName]="doSomething"> ... </a>
```